src/cdomains/exp.ml

```diff
@@ -290,19 +294,22 @@ struct
     in
     let rec helper exp =
       match exp with
+      | BinOp ((PlusPI | IndexPI), e1, e2, _) ->
+        helper e1 (* TODO: what about e2? add to some Index offset to all? *)
```

Comment:
Sam here, how confident are we that we are doing something sensible for pointer arithmetic?